### PR TITLE
istep 8.1 & 8.2 swap  and enabling proc_select_boot_master in BMC

### PR DESCRIPTION
**Pull request by lkammath**
Changes:
   istep 8.1 and 8.2 is swapped as per IPL flow document v0.91.00
   Enabled proc_select_boot_master in BMC

Signed-off-by: Lakshminarayana R. Kammath <lkammath@in.ibm.com>

### CODE REVIEW

**Comment by lkammath** [comment]
@ojayanth @thejta

**Comment by lkammath** [comment]
> Split into two commits/PR
> 
>     1. enable proc_select_boot master. --
> 
>     2. version 0.91 changes --- This required ipl/hostboot updates.

Done

File: src/istep/edbgIstep.C [hunk 1]
```diff
@@ -66,7 +66,7 @@ const edbgIPLTable::edbgIStep_t edbgIPLTable::cv_edbgIStepTable[] =
   { 0,   8,                             "proc_prep_ipl",                   EDBG_ISTEP_NOOP},
   { 0,   9,                               "edramrepair",                   EDBG_ISTEP_NOOP},
   { 0,  10,                          "asset_protection",                   EDBG_ISTEP_NOOP},
-  { 0,  11,                   "proc_select_boot_master",                   EDBG_ISTEP_NOOP},
+  { 0,  11,                   "proc_select_boot_master",                   EDBG_ISTEP_BMC},
```
Comment (ojayanth) [inline src/istep/edbgIstep.C:69]: This is good fro merge.

File: src/istep/edbgIstep.C [hunk 2]
```diff
@@ -150,8 +150,8 @@ const edbgIPLTable::edbgIStep_t edbgIPLTable::cv_edbgIStepTable[] =
   { 7,   3,                                  "mss_freq",                   EDBG_ISTEP_HOST},
   { 7,   4,                            "mss_eff_config",                   EDBG_ISTEP_HOST},
   { 7,   5,                           "mss_attr_update",                   EDBG_ISTEP_HOST},
-  { 8,   1,                     "host_slave_sbe_config",                   EDBG_ISTEP_HOST},
-  { 8,   2,                            "host_setup_sbe",                   EDBG_ISTEP_HOST},
+  { 8,   1,                            "host_setup_sbe",                   EDBG_ISTEP_HOST},
```
Comment (ojayanth) [inline src/istep/edbgIstep.C:153]: Need to hold the merge till IPL/Hostboot support is ready

**Comment by lkammath** [comment]
@ojayanth is this OK to integrate?

**Comment by ojayanth** [comment]
> @ojayanth is this OK to integrate?



> @ojayanth is this OK to integrate?
Host update is in place?

**Comment by lkammath** [comment]
> > @ojayanth is this OK to integrate?
> 
> > @ojayanth is this OK to integrate?
> > Host update is in place?
Yes
https://github.ibm.com/open-power/hostboot/blob/master-p10/src/include/usr/isteps/istep08list.H

**Comment by ojayanth** [comment]
Approved.

**Comment by lkammath** [comment]
@thejta this can be integrated now

**Pull request merged**